content/bn/docs/reference/glossary/cluster-architect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ক্লাস্টার আর্কিটেক্ট
+id: cluster-architect
+date: 2018-04-12
+full_link: 
+short_description: >
+  একজন ব্যক্তি যিনি এক বা একাধিক কুবের্নেটিস ক্লাস্টারের সাথে সম্পর্কিত পরিকাঠামো ডিজাইন করেন।
```

Comment:
@Noblesse013 

Please use `ইনফ্রাস্ট্রাকচার` instead of `পরিকাঠামো` as this term is used very often.